doc/nrf/libraries/caf/sensor_data_aggregator.rst

```diff
@@ -0,0 +1,80 @@
+.. _caf_sensor_data_aggregator:
+
+CAF: Sensor data aggregator module
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |sensor_data_aggregator| of the :ref:`lib_caf` (CAF) is a simple module responsible for aggregating sensor data in form of ``sensor_event`` and passing them further in packages.
+It can be used in both single-core and multi-core SoCs.
+In multi-core SoCs using the |sensor_data_aggregator| can reduce power consumption.
+When used with multi-core SoCs, one core gathers data from sensors and when there is sufficient data to analyze, the first core wakes up the other core and sends the aggregated data to that core.
```

Comment:
```suggestion
It can be used in both single-core and multi-core SoCs.

When used with multi-core SoCs, the |sensor_data_aggregator| can reduce power consumption.
One core gathers data from sensors and when there is sufficient data to analyze, the first core wakes up the other core and sends the aggregated data to that core.
```